lib.php

```diff
@@ -189,7 +189,8 @@ function local_pages_extend_navigation(global_navigation $nav) {
  *
  * @return array
  */
-function local_pages_get_fontawesome_icon_map() {
+function local_pages_get_fontawesome_icon_map()
+{
```

Comment:
There is no reason to change the code style.
Always respect the original code style if it's a 3rd party plugin.
Unless you have been ask to refactor the whole plugin.

```suggestion
function local_pages_get_fontawesome_icon_map(): array {
```